types/src/on_chain_config/aptos_features.rs

```diff
@@ -248,6 +249,7 @@ impl FeatureFlag {
             FeatureFlag::ENABLE_LAZY_LOADING,
             FeatureFlag::MONOTONICALLY_INCREASING_COUNTER,
             FeatureFlag::ENABLE_TRUSTED_CODE,
+            FeatureFlag::ENABLE_ENUM_OPTION,
```

Comment:
We only set `ENABLE_ENUM_OPTION` to default so that devnet can directly use updated option module

Reply:
See above, default is to NOT OVERRIDE. (Default is the final behavior where we want to land.)

Reply:
done